### PR TITLE
[hotfix][docs] Shortcode syntax

### PR DESCRIPTION
**Pull request by tisonkun**
This is a follow-up to https://github.com/apache/flink-connector-pulsar/pull/14.

cc @xintongsong @syhily

### CODE REVIEW

**Review by xintongsong — APPROVED** [review]
LGTM

**Comment by syhily** [comment]
I think we may need this syntax?

File: docs/content.zh/docs/connectors/datastream/pulsar.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Flink 当前提供 [Apache Pulsar](https://pulsar.apache.org) Source 和 Sink 
 
 当前支持 Pulsar 2.10.0 及其之后的版本，建议在总是将 Pulsar 升级至最新版。如果想要了解更多关于 Pulsar API 兼容性设计，可以阅读文档 [PIP-72](https://github.com/apache/pulsar/wiki/PIP-72%3A-Introduce-Pulsar-Interface-Taxonomy%3A-Audience-and-Stability-Classification)。
 
-{{< artifact flink-connector-pulsar 4.0.0-SNAPSHOT >}}
+{{< artifact flink-connector-pulsar >}}
```
Comment (syhily) [inline docs/content.zh/docs/connectors/datastream/pulsar.md:33]:
```suggestion
{{< connector_artifact flink-connector-pulsar 4.0.0-SNAPSHOT >}}
```

File: docs/content/docs/connectors/datastream/pulsar.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Flink provides an [Apache Pulsar](https://pulsar.apache.org) connector for readi
 You can use the connector with the Pulsar 2.10.0 or higher. It is recommended to always use the latest Pulsar version.
 The details on Pulsar compatibility can be found in [PIP-72](https://github.com/apache/pulsar/wiki/PIP-72%3A-Introduce-Pulsar-Interface-Taxonomy%3A-Audience-and-Stability-Classification).
 
-{{< artifact flink-connector-pulsar 4.0.0-SNAPSHOT >}}
+{{< artifact flink-connector-pulsar >}}
```
Comment (syhily) [inline docs/content/docs/connectors/datastream/pulsar.md:34]:
```suggestion
{{< connector_artifact flink-connector-pulsar 4.0.0-SNAPSHOT >}}
```

**Review by syhily — CHANGES_REQUESTED** [review]
@xintongsong Should we change the shortcode like this?

**Comment by xintongsong** [comment]
I'd be fine with either way as long as it fix the documentation build.

**Comment by syhily** [comment]
Sorry for the inconvenience to the documentation building. I will check more closely on similar errors.

**Comment by syhily** [comment]
@tisonkun XComp has created a new PR https://github.com/apache/flink-connector-pulsar/pull/22 for fixing the doc problem. We can close this PR now.

**Pull request closed without merging**